### DISCUSSION TITLE
Home Mini just installed

### DISCUSSION THREAD

**Discussion by TheNeutrinoRaged** [BottlecapDave/HomeAssistant-OctopusEnergy#343] [Q&A] [answered]
Hi, I've been sent a Home Mini and just installed it about 20 minutes ago.

Is there anything I need to do to get it working with the integration? or just sit tight and wait for it to work on it's own in the background?

Thanks

**Comment by TheNeutrinoRaged** [comment, marked as answer]
Just read the setup guide again (should've done that before posting really), ticked the box in configuration and the extra sensors have shown up

**Comment by deltamelter** [comment]
@TheNeutrinoRaged How long since you requested?

**Reply by TheNeutrinoRaged** [reply]
Not sure now I can't find an email confirming my request, but I switched to octopus in Feb 2023 and I'm sure I requested it pretty soon after joining.
